Dockerfile

```diff
@@ -10,7 +10,8 @@ RUN apt-get update && apt-get install --no-install-recommends -y \
 	&& for ver in \
 		#2.051 2.052 2.053 2.054 2.055 2.056 2.057 2.058 2.059 \
 		2.060 2.061 2.062 2.063 2.064 2.065.0 2.066.0 2.067.1 2.068.2 2.069.2 \
-		2.070.2 2.071.2 2.072.2 2.073.2 2.074.1 2.075.1 2.076.1 2.077.1 2.078.1 2.079.1 2.080.1 2.081.1 \
+		2.070.2 2.071.2 2.072.2 2.073.2 2.074.1 2.075.1 2.076.1 2.077.1 2.078.1 2.079.1 \
+		2.080.1 2.081.1 2.081.2 2.082.0 2.082.1 2.083.0 2.083.1 2.084.0 \
```

Comment:
This image is already gigantic in size, let's please only use the last stable release of every major release, that's reasonable good enough and doesn't strain the limited resources.

Reply:
:+1: dully noted.

Reply:
https://github.com/dlang-tour/core-dreg/pull/6